playwright.config.ts

```diff
@@ -72,6 +72,14 @@ const config: PlaywrightTestConfig = {
         headless: true,
       },
     },
+    {
+      name: 'global',
```

Comment:
Nit: might be helpful to document what goes into global category ~ wdyt?

Reply:
added, thank you!